src/committed-message.ts

```diff
@@ -49,7 +50,7 @@ export abstract class CommittedMessage implements Nullable {
   }
 
   payload(): string {
-    return this.commit.body.trim()
+    return new CommitBody(this.commit.body).getPayload().trim()
```

Comment:
@yeraydavidrodriguez now that we use a JSON the trim should be done inside CommitBody in case we continue doing it.

Reply:
I will remove it.